draft/overlay/RealTimeHAT.md

```diff
@@ -0,0 +1,83 @@
+<!--
+---
+name: RealTimeHAT
+class: board
+type: multi
+formfactor: HAT
+manufacturer: InnoRoute GmbH
+description: A sblabalbal board that can be used easily to show status of the Pi
+github: https://github.com/InnoRoute/RealtimeHAT
+image: 'RealTimeHAT.png'
+pincount: 40
+eeprom: yes
+power:
+  '1':
+  '2':
+  '4':
+ground:
+  '6':
+  '9':
+  '14':
+  '17':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+pin:
+  '3':
+    mode: i2c
+  '5':
+    mode: i2c
+  '7':
+    name: PHY_MDC
+  '11':
+    name: PHY_MDIO
+  '13':
+    name: FPGA_Interrupt
+    direction: Input
+  '15':
+    name: Buffer_Full
+    direction: Input
+  '19':
+    mode: spi
+  '21':
+    mode: spi
+  '23':
+    mode: spi
+  '24':
+    mode: spi
+  '26':
+    mode: spi
+  '27':
+    mode: i2c
+  '28':
+    mode: i2c
+i2c:
+  '0x50':
+    name: ID EEPROM
+    device: I2C0 - ID EEPROM
+  '0x24':
+    name: PMIC
+    device: I2C1 - PMIC
+  '0x43':
+    name: IO Expender
+    device: I2C1 - IO Expander
+-->
+# RealTimeHAT
+
+The RealTimeHAT extends the Gigabit-Ethernet interface of the Raspberry Pi by professional networking functions.
+Typical applications are **industrial real-time communication** and **network monitoring**.
+
+The HAT provides **3 Gigabit-Ethernet ports**, one of which connecting to the Raspberry Pi's Ethernet port.
+Connection to the Raspberry Pi is realized via the GPIO connector and via a (short) Ethernet cable. Interface SPI0 is for main configuration, using either CE0# or CE1#. Beyond the standard interfaces, the HAT uses 4 dedicated pins
```

Comment:
I guess it's not really clear what "Beyond the standard interfaces" means? :shrug:

Reply:
right, this sentence is not necessary users can simply read the pinout